File: 2608-count-the-digits-that-divide-a-number/count-the-digits-that-divide-a-number.py
class Solution:
    def countDigits(self, num: int) -> int:
        result = 0
        for i in range(len(str(num))):
            str_num = str(num)
            if(num % int(str_num[i]) == 0 ):
                result +=1
        return(result)

                

File: 2608-count-the-digits-that-divide-a-number/count-the-digits-that-divide-a-number.cs
public class Solution {
    public int CountDigits(int num) {
        var strDigit = Convert.ToString(num);
			var result = 0;
			for (int i = 0; i < strDigit.Length; i++)
			{
				var temp = strDigit[i] - '0';
				if (num % temp == 0)
				{
					result++;
				}
				
			}
		
			return(result);
        
    }
}